src/components/Main.vue

```diff
@@ -120,7 +120,8 @@ export default {
           .replace(/Kat/, 'KAT')
           .replace(/Wob/, 'WOB')
           .replace(/Ta/, 'TA')
-          .replace(/ ?Plus/g, '+');
+          .replace(/ ?Plus/g, '+')
+          .replace(/ ?Dot/g, '.');
```

Comment:
I want to see if we can work out a better solution for this. For reasons I have yet to work out, attempting to capture the space after `Dot` prevents the UI from fully loading. I'm starting to wonder if it wouldn't be better to just assign the keyset name as a string in the `.js` file and load it directly, instead of manipulating it via regexes.

@yanfali Thoughts?

Reply:
There's no way for me to play around with this is there? Not easily anyway?

Reply:
It's probably because it's a regex and '.' is actually a match all character. You might look at some unicode replacements that don't trigger the regex.

Reply:
`.` ?

Reply:
Normal Period  -`Fro.Yo`

"One Dot Leader" -`Fro․Yo`

Reply:
Today I learned that Firefox, Chrome and Brave all will still parse escaped characters in a `.replace` regex. yanfali's comment about `.` being the regex match all character prompted me to try escaping it, and it works.

```suggestion
          .replace(/ ?Dot ?/g, '\.');
```